OmiseSDK/SourceType.swift

```diff
@@ -0,0 +1,201 @@
+import Foundation
+
+public enum SourceType {
+    case internetBankingBAY
+    case internetBankingKTB
+    case internetBankingSCB
+    case internetBankingBBL
+    
+    case installmentBAY
+    case installmentFirstChoice
+    case installmentBBL
+    case installmentKTC
+    case installmentKBank
+    case installmentSCB
+    
+    case alipay
+    case barcodeAlipay
+    case billPaymentTescoLotus
+    case econtext
+    case promptpay
+    case paynow
+    case truemoney
+    case pointsCiti
+    
+    case other(value: String)
+    
+    var value: String {
```

Comment:
What would you think about conforming to `RawRepresentable` and use `rawValue` instead of `value`?

Reply:
I use `value` just about the naming and want to call it like a short name. It's ok if we conform `RawRepresentable` protocol. But how about if we still use `value` property and return the value with 
```
var value: String {
return self.rawValue
}
```
I know it is the same value with `rawValue`, what do you think? 😄

Reply:
I would just rename `value` to `rawValue`. I just had the feeling it's maybe better for understanding to use the default protocols and the naming `rawValue` is maybe also more clear than `value`. But when you prefer `value` it's also okay for me. :)

Reply:
Your reason is sound good to me too, let me revise a moment. 👍

Reply:
done 🚀